tests/test_travis_mf_multiplePS/error_mf.cpp

```diff
@@ -0,0 +1,97 @@
+#include <Tasks.h>
```

Comment:
I think this is the same code as the one in cirrus/tests/test_travis_mf. We can just use that no?